scripts/init_replica.sh

```diff
@@ -174,6 +174,13 @@ echo "  port:" >> mongod.conf
 if [ "$version" == "3.6" ] || [ "$version" == "4.0" ] || [ "$version" == "4.2" ] || [ "$version" == "5.0" ] || [ "$version" == "6.0" ]; then
     echo "  bindIpAll: true" >> mongod.conf
 fi
+echo "  tls:" >> mongod.conf
+echo "    mode: requireTLS" >> mongod.conf
+echo "    CAFile: /etc/mongodb/ssl/ca.crt" >> mongod.conf
```

Comment:
How can we provide the `/etc/mongodb/ssl/ca.crt`?
Is it a input option when launching CloudTemplate or data in Secrets Manager?

Reply:
Oh, I do remember we talked about this, in which we let the users choose if they want to create a replica set with a specific certificate and key.